README.md

```diff
@@ -266,21 +266,20 @@ We can see we need to define a `Snumberp` and a `Fnumberp`. `Qt` and
 ``` rust
 // This is the function that gets called when 
 // we call numberp in elisp.
-fn Fnumberp(object: LispObject) -> LispObject {
+fn numberp(object: LispObject) -> LispObject {
     if lisp::NUMBERP(object) {
-        unsafe {
-            Qt
-        }
+        LispObject::constant_t()
     } else {
-        Qnil
+        LispObject::constant_nil()
     }
 }
 
 // This defines a built-in function in elisp, which is a represented
 // with a static struct.
 defun!("numberp", // the name of our elisp function
-       Fnumberp, // the rust function we want to call
+       Fnumberp, // the function that will be called by C (this calls numberp).
```

Comment:
Why isn't this `Fnumberp(object)`? It's not quite obvious to me when we need `Ffoo` vs `Ffoo(bar)` in the `defun!` declaration, would you mind clarifying in the docstring?